src/test/cypress/support/pageobjects/ExamCreationPage.ts

```diff
@@ -0,0 +1,102 @@
+import dayjs from 'dayjs';
+import { dayjsToString } from '../utils';
+
+/**
+ * A class which encapsulates UI selectors and actions for the exam creation page.
+ */
```

Comment:
It would be nice to add the path to the described page here so its easier to tell which page you mean.

Reply:
Makes sense. I will update the documentation in the [followup branch](https://github.com/ls1intum/Artemis/tree/test/cypress%2Fexam-participation)